thingif/src/main/java/com/kii/thingif/TraitThingIFAPI.java

```diff
@@ -553,6 +560,75 @@ public static String getSDKVersion() {
         return ThingIFAPI.getSDKVersion();
     }
 
+    /**
+     * Post new command to IoT Cloud.
+     * Command will be delivered to specified target and result will be notified
+     * through push notification.
+     * @param form form of command. It contains name of schema, version of
+     * schema, list of actions etc.
+     * @return Created Command instance. At this time, Command is delivered to
+     * the target Asynchronously and may not finished. Actual Result will be
+     * delivered through push notification or you can check the latest status
+     * of the command by calling {@link #getCommand}.
+     * @throws ThingIFException Thrown when failed to connect IoT Cloud Server.
+     * @throws ThingIFRestException Thrown when server returns error response.
+     */
+    @NonNull
+    @WorkerThread
+    public TraitCommand postNewCommand(TraitCommandForm form) throws ThingIFException {
+        //TODO: implement me
+        return null;
+    }
+
+    /**
+     * Get specified command.
+     * @param commandID ID of the command to obtain. ID is present in the
+     *                  instance returned by {@link #postNewCommand}
+     *                  and can be obtained by {@link TraitCommand#getCommandID}
+     *
+     * @return Command instance.
+     * @throws ThingIFException Thrown when failed to connect IoT Cloud Server.
+     * @throws ThingIFRestException Thrown when server returns error response.
+     * @throws UnsupportedSchemaException Thrown when the returned response has a schema that cannot handle this instance.
+     * @throws UnsupportedActionException Thrown when the returned response has a action that cannot handle this instance.
+     */
+    @NonNull
+    @WorkerThread
+    public TraitCommand getCommand(
```

Comment:
`TraitThingIFAPI.getCommand` should only receive trait format command. And `AbstractCommand` is package private, not available to outside, because it doesn't hold `actions` and `actionResults` attributes.